rackcity/utils/change_planner_utils.py

```diff
@@ -75,22 +78,44 @@ def detect_conflicts_cp(sender, **kwargs):
     ).update(asset_conflict_hostname=asset)
 
     # asset rack location conflicts with an active assetCP
-    asset.location_conflict.clear()
-    for assetcp in AssetCP.objects.filter(
-        Q(rack=asset.rack_id)
-        & ~Q(related_asset=asset.id)
-        & Q(change_plan__execution_time=None)
-    ):
-        try:
-            validate_asset_location(
-                asset.rack_id,
-                assetcp.rack_position,
-                assetcp.model.height,
-                asset_id=assetcp.id,
-            )
-        except LocationException:
-            assetcp.asset_conflict_location = asset
-            AssetCP.objects.filter(id=assetcp.id).update(asset_conflict_location=asset)
+
+    if asset.model.is_rackmount():
+        asset.location_conflict.clear()
```

Comment:
small nit but move this line outside of if statement bc we need it in both cases

Reply:
Will do